openedx/features/assessment/tests/test_helpers.py

```diff
@@ -0,0 +1,247 @@
+from datetime import datetime
+
+import mock
+from django.contrib.auth.models import User
+from pytz import UTC
+from submissions.models import Submission
+
+from openedx.features.assessment import helpers
+from openedx.features.assessment.tests.factories import AnonymousUserIdFactory, StudentItemFactory, SubmissionFactory
+from student.tests.factories import CourseEnrollmentFactory
+from xmodule.modulestore.django import modulestore
+from xmodule.modulestore.tests.django_utils import ModuleStoreTestCase
+
+from .constants import (
+    FOUR_DAYS_AGO,
+    PHILU_BOT_NAME,
+    TEST_RUBRIC_CRITERIA,
```

Comment:
`TEST_RUBRIC_CRITERIA` is not used

Please run isort on all files

Reply:
done